addons/website/views/snippets/s_mega_menu_big_icons_subtitles.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
 <template id="s_mega_menu_big_icons_subtitles" name="Menu - Big icons &amp; subtitles" groups="base.group_user">
-    <section class="s_mega_menu_big_icons_subtitles pt24 pb24" data-oe-shape-data="{'shape':'web_editor/Zigs/03','flip':[]}">
+    <section class="s_mega_menu s_mega_menu_big_icons_subtitles pt24 pb24" data-oe-shape-data="{'shape':'web_editor/Zigs/03','flip':[]}">
```

Comment:
We should not need this. Besides, our convention for s_ classes is that `s_xxx_yyy` is either:
- A class put on a snippet root to uniquely identify it (like `s_mega_menu_big_icons_subtitles` here)
- A class put on a snippet's child which has an ancestor with the class `s_xxx`